test/test_KubeLibrary.py

```diff
@@ -678,3 +694,53 @@ def test_get_cron_job_details_in_namespace(self, mock_lnp):
         kl = KubeLibrary(kube_config='test/resources/k3d')
         cron_job_details = kl.get_cron_job_details_in_namespace('hello', 'default')
         self.assertEqual('mytestlabel', cron_job_details.items.metadata.labels.TestLabel)
+
+    @mock.patch('kubernetes.client.RbacAuthorizationV1Api.delete_namespaced_role')
+    def test_delete_role_in_namespace(self, mock_lnp):
```

Comment:
You could test handling situation when there is no role to remove. Other than that the mocked response is wrong because as I understand it will return V1 status.